target_postgres/json_schema.py

```diff
@@ -1,5 +1,14 @@
 import re
 
+from jsonschema import Draft4Validator
+from jsonschema.exceptions import SchemaError
+
+
+class Error(Exception):
```

Comment:
Can we call this something like `JSONSchemaException`? `Error` is very generic.

Reply:
I'm up for that. I tend to prefer not double namespacing vars (ie, the namespace is _already_ `json_schema` so to get to this you already have to do `json_schema.Error`).

This will be a common pattern in the onslaught of prs I have up, I'll tidy this up.

Reply:
@AlexanderMann It's just more common in python to have a distinct name, it also makes the stack trace clearer